endpoints/cookie_sync.go

```diff
@@ -192,26 +198,22 @@ func (c *cookieSyncEndpoint) writeParseRequestErrorMetrics(err error) {
 	}
 }
 
-func (c *cookieSyncEndpoint) setLimit(request cookieSyncRequest) (cookieSyncRequest, error) {
-	if request.Account != "" {
-		accountInfo, errs := accountService.GetAccount(context.Background(), c.config, c.accountsFetcher, request.Account)
-		if len(errs) > 0 {
-			return request, combineErrors(errs)
-		}
+func (c *cookieSyncEndpoint) setLimit(request cookieSyncRequest, account *config.Account) cookieSyncRequest {
```

Comment:
Consider passing in the `account.CookieSync` as an argument instead of the entire `account`.